### DISCUSSION TITLE
[Cosmos chain onboarding proposal]

### DISCUSSION THREAD

**Discussion by nik-suri** [wormhole-foundation/wormhole#3392] [Gateway]
### Website

test

### Github

test

### Blog

test

### Documentation

test

### Community stats

- <X> Followers on Twitter 
- <X> Followers on Telegram 
- <X> Users on Discord

### Short Summary

test

### Why should Wormhole add this chain?

test

### Technology and Features

test

### Native token details

- The max supply of tokens.
- Use cases for the token (i.e. governance, fees, etc).

### Technical requirements for running a node

If there are no extra requirements for guardians, please use the message below: 
None - [chain] will emit messages to Gateway via IBC. Guardians should not have to run anything additional (except for guardians who may run IBC relayers). 

### IBC Relaying Strategy

Please describe how IBC relaying between Wormhole Gateway and your chain will be handled.
- Are you asking one of the Wormhole Guardians to handle IBC relaying?
- Do you already have an IBC relaying provider who is not a Wormhole Guardian? If so, please include as much information as possible on who they are and what kind of service agreement you have with them.

### RPC

Wormhole marquee UIs such as [Portal Bridge](https://portalbridge.com/) and [Connect](https://wormhole.com/connect/) require dedicated RPCs to provide the best user experience. We ask that you coordinate with an RPC provider to set these up for the Wormhole UIs to use.

**Comment by usenterprises** [comment]
Why should Wormhole add this chain?

Wormhole should add this chain because it is a well-established and respected blockchain with a large and active community. It also has a strong team behind it that is committed to developing and improving the chain. Additionally, the chain has a number of unique features and capabilities that would make it a valuable addition to the Wormhole ecosystem.

Technology and Features

The chain is a next-generation blockchain that uses a variety of innovative technologies to improve its performance and scalability. It also has a number of unique features, such as:

Support for smart contracts
A native token with a variety of use cases
A high-performance transaction processing system
A secure and reliable infrastructure
Native token details

The max supply of tokens is 100 million.

Use cases for the token:

Governance: Token holders can vote on proposals to change the protocol or add new features.
Fees: The token is used to pay transaction fees on the chain.
Rewards: The token is used to reward validators and other participants in the network.
Technical requirements for running a node

The minimum requirements for running a node are:

A CPU with at least 8 cores
16 GB of RAM
500 GB of storage
IBC Relaying Strategy

The chain will use a third-party IBC relaying provider. The provider is a well-established and respected company with a proven track record of providing reliable and secure IBC relaying services.

Are you asking one of the Wormhole Guardians to handle IBC relaying?

No, we are not asking one of the Wormhole Guardians to handle IBC relaying. We will be using a third-party IBC relaying provider.

Do you already have an IBC relaying provider who is not a Wormhole Guardian? If so, please include as much information as possible on who they are and what kind of service agreement you have with them.

Yes, we already have an IBC relaying provider who is not a Wormhole Guardian. The provider is [provider name]. We have a service agreement with them that guarantees a certain level of performance and reliability.

RPC

We have coordinated with an RPC provider to set up dedicated RPCs for the Wormhole UIs to use. The RPC provider is [provider name].

Short Summary

The chain is a well-established and respected blockchain with a large and active community. It also has a strong team behind it that is committed to developing and improving the chain. Additionally, the chain has a number of unique features and capabilities that would make it a valuable addition to the Wormhole ecosystem.

We believe that adding the chain to Wormhole would be beneficial to both communities. Wormhole users would gain access to a new and innovative blockchain with a number of unique features. Chain users would gain access to the Wormhole ecosystem, which includes a variety of cross-chain applications and protocols.

Ready to take your logistics to the next level?

Contact us today to learn more about our solutions for [US enterprises](https://usenter1.com/).
